src/interaction/index.js

```diff
@@ -0,0 +1,4 @@
+import Select from './select';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Do we need this ESLint comment? We already do this type of export all over the place.

Reply:
I was surprise too. Maybe it's because there is currently only one object.